cypress/support/step_definition/course/editCourse/editCourse.spec.js

```diff
@@ -0,0 +1,52 @@
+import Edit_Course from '../../../pages/course/pageEditCourse'
+import { And, When } from 'cypress-cucumber-preprocessor/steps'
+
+const editCourse = new Edit_Course()
+
+And('I click on course', () => {
+    editCourse.clickOnCourse()
+})
+
+Then('I can see course page', () => {
```

Comment:
This step can be added to the other course tests (delete now, edit later) as well.

Reply:
Yes, I think so. But then we should move it to a common file.

Reply:
Agreed.

Reply:
As d3n153 created a pageCommon.js in the branch added new school administration, these changes should be done after merging both branches.